clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/ops/LoadBalancerStatusAware.groovy

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016 Target, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.openstack.deploy.ops
+
+import com.netflix.spinnaker.clouddriver.openstack.client.BlockingStatusChecker
+import com.netflix.spinnaker.clouddriver.openstack.config.OpenstackConfigurationProperties
+import com.netflix.spinnaker.clouddriver.openstack.deploy.exception.OpenstackProviderException
+import com.netflix.spinnaker.clouddriver.openstack.security.OpenstackCredentials
+import org.openstack4j.model.network.ext.LbProvisioningStatus
+import org.openstack4j.model.network.ext.LoadBalancerV2
+
+trait LoadBalancerStatusAware {
```

Comment:
Are you planning on replacing other areas that could use this too?  (i.e. Upsert/Delete LB operations)  If not, then add TODO.